changelog/Microsoft.CPIM.json

```
@@ -0,0 +1,30 @@
+{
+  "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "ea97a289-11cc-4238-8c64-66ea732ad207",
+          "ApiChange": "Enumeration",
+          "ChangedApiName": "identityProviderState",
+          "ChangeType": "Addition",
+          "Description": "Added the **identityProviderState** enumeration type.",
+          "Target": "identityProviderState"
+        },
+        {
+          "Id": "ea97a289-11cc-4238-8c64-66ea732ad207",
+          "ApiChange": "Property",
+          "ChangedApiName": "state",
+          "ChangeType": "Addition",
+          "Description": "Added the `state` property to the [builtInIdentityProvider](https://docs.microsoft.com/en-us/graph/api/resources/builtInIdentityProvider?view=graph-rest-beta) resource.",
```

Recommended enhancement.
```suggestion
          "Description": "Added the **state** property to the [builtInIdentityProvider](https://docs.microsoft.com/en-us/graph/api/resources/builtInIdentityProvider?view=graph-rest-beta) resource. Use this property to enable or disable the Microsoft accounts built-in identity provider in your tenant.",
```